pkg/crds/crds_test.go

```diff
@@ -39,14 +47,28 @@ var _ = Describe("test crds pkg", func() {
 		}, 50)
 	})
 	Context("GetOperatorCRDSource", func() {
+		BeforeEach(func() {
+			_ = getOperatorCRDSource(opv1.Calico)
+			_ = getOperatorCRDSource(opv1.TigeraSecureEnterprise)
+		})
 		Measure("should quickly load operator source CRDs", func(b Benchmarker) {
 			runtime := b.Time("runtime", func() {
-				_ = getEnterpriseCRDSource()
+				_ = getOperatorCRDSource(opv1.Calico)
+			})
+			Expect(runtime.Seconds()).Should(BeNumerically("<", 0.2), "loading Calico operator CRDs shouldnt take too long.")
+		}, 50)
+		Measure("should quickly load Enterprise operator source CRDs", func(b Benchmarker) {
+			runtime := b.Time("runtime", func() {
+				_ = getOperatorCRDSource(opv1.TigeraSecureEnterprise)
 			})
-			Expect(runtime.Seconds()).Should(BeNumerically("<", 0.2), "loading operator CRDs shouldnt take too long.")
+			Expect(runtime.Seconds()).Should(BeNumerically("<", 0.2), "loading Enterprise operator CRDs shouldnt take too long.")
```

Comment:
With the changes in this PR, locally I was able to turn this down to `0.02s` so the value being used here is already 10x what works locally. Is it a useful to check that the performance is not 100x worse?
I think we've only had occasional flakes with 0.2s, would we not want to see if we see this flake still pop up with the additional priming of the test? We can easily bump this in the future. But I doubt we'd ever have reason to turn the value down.

Reply:
idk. To be honest I am not sure what the value of this test even really is, so perhaps I am not the right person to review :D 

If it's actually down to .02 then a 10x buffer seems fine. But then again, are we even testing the same thing if we're pre-loading the CRDs? What's the intention of these tests?

Reply:
IIRC I added the tests because I was concerned about loading our large CRDs on each reconcile and I wanted us to be aware if the loading time of them started to creep up and then we could put in some kind of caching at that point.

Reply:
Hmm ok I already made the CRDs cached. So I agree there is little reason to benchmark this.